src/app/CommandHandler.cpp

```diff
@@ -470,8 +470,7 @@ CHIP_ERROR CommandHandler::AddStatus(const ConcreteCommandPath & aCommandPath, c
     return AddStatusInternal(aCommandPath, StatusIB(aStatus));
 }
 
-CHIP_ERROR CommandHandler::AddStatusAndLogIfFailure(const ConcreteCommandPath & aCommandPath, const Status aStatus,
-                                                    const char * aMessage)
+void CommandHandler::AddStatusAndLogIfFailure(const ConcreteCommandPath & aCommandPath, const Status aStatus, const char * aMessage)
```

Comment:
if void, the "And log if failure" in the name seems not needed. We do not name all our functions that log something with "and log if failure".

Consider renaming to just AddStatus as I imagine void functions would log if things don't go their way.

Reply:
Yeah :(